### ISSUE TITLE
Improve Bloom

### ISSUE THREAD

**Issue by dmnsgn** [pex-gl/pex-shaders#11]
- [ ] add tint color
- [ ] anamorphic ratio
- [ ] lens dirt
- [ ] pack bloom intensity in bloom texture alpha for selective bloom

**Comment by dmnsgn** [comment]
> * [ ]  combine 8 upscale passes adding all blur steps into threshold texture into one final blend pass

Should we also add a blending mode option for this (if we want to blend with screen for instance)?

**Comment by dmnsgn** [comment]
- [ ] use bicubic sampling for smooth bloom
- [ ] or use use mipmaps directly (WebGL2) instead of rendering to 8 textures

**Comment by vorg** [comment]
[ ] add together in one pass instead of 8

**Comment by vorg** [comment]
Are we using Kawase technique here? Very good video on modern bloom that talks about downsampling but also upsampling instead of just blending https://www.youtube.com/watch?v=ml-5OGZC7vE

**Comment by dmnsgn** [comment]
> Are we using Kawase technique here? Very good video on modern bloom that talks about downsampling but also upsampling instead of just blending https://www.youtube.com/watch?v=ml-5OGZC7vE

No, our downsample is 4x4 box filter (+Karis's luma weighted average). No Kuwase, no 13 taps.
